docker-compose.yml

```diff
@@ -1,5 +1,14 @@
 version: "3"
 services:
+  app:
+    container_name: app
+    build: .
+    command: bash -c "python manage.py makemigrations && python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```

Comment:
If you have more than 1 command it's better to have them in `entrypoint.sh` file with appropriate linting. It would be easier to maintain and read